### PR TITLE
Add net.krafting.Playlifin

### PR DESCRIPTION
**Pull request by Krafting**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by Krafting** [comment]
bot, build net.krafting.Playlifin

**Comment by Krafting** [comment]
bot, build net.krafting.Playlifin

**Comment by Krafting** [comment]
bot, build net.krafting.Playlifin

File: net.krafting.Playlifin.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  -  --socket=fallback-x11
+
+modules:
+  - name: Playlifin
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 Playlifin.py /app/bin/Playlifin.py
+      - install -Dm755 scripts/sync_playlist.py /app/bin/scripts/sync_playlist.py
+      - install -Dm 0644 net.krafting.Playlifin.desktop  /app/share/applications/net.krafting.Playlifin.desktop
+      - install -Dm 0644 net.krafting.Playlifin.metainfo.xml  /app/share/metainfo/net.krafting.Playlifin.metainfo.xml
```
Comment (bbhtt) [inline net.krafting.Playlifin.yml:20]:
```
ap-util validate net.krafting.Playlifin.metainfo.xml 
net.krafting.Playlifin.metainfo.xml: FAILED:
• style-invalid         : <caption> cannot end in '.' [The main Playlifin window.]
• style-invalid         : <caption> cannot end in '.' [The main window with logs.]
Validation of files failed
```

**Comment by Krafting** [comment]
bot, build net.krafting.Playlifin

**Pull request closed without merging**